src/recipe/Baked.Recipe.Service.Application/Ui/UiLayer.cs

```diff
@@ -9,19 +9,37 @@ namespace Baked.Ui;
 
 public class UiLayer : LayerBase<GenerateCode>
 {
+    public delegate string NewLocaleKey(string key, string? value = default);
```

Comment:
i didn't see anywhere using this `value` optional. i think we need to remove this?